Add retry and timeout to http template handler (#1145)
Sceptre/sceptre: docs/_source/docs/template_handlers.rst

http
~~~~~~~~~~~~~

Downloads a template from a url on the web.  By default, this handler will attempt to download

> templates with 5 retries and a download timeout is set to 5 seconds.  The retry and timeout

Reviewer: ```suggestion
templates with 5 retries and a download timeout of 5 seconds.  The retry and timeout
```

Author: fixed
